src/Map.js

```diff
@@ -159,7 +159,7 @@ export function Map(props){
   		}
 
       // zoom to marker bounds, plus padding (percentage)
-      map.current.fitBounds(markers.current.getBounds().pad(0.5));
+      map.current.fitBounds(markers.current.getBounds().pad(0.5));  
```

Comment:
Spurious whitespace

Reply:
Yeah - sorry about that.  I see it is a problem elsewhere too (App.scss).  I will try to clean it up before the merge.

Reply:
Whitespace removed